content/post/2020-01-02-cran-error/index.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+slug: cran-error
+title: "From Shock to Competence: How Not to Panic When You Receive E-mail from CRAN about Failed Checks"
+authors:
+  - Julia Romanowska
+date: "`r Sys.Date()`"
```

Comment:
Eventually we'll change it to publication date (I'm guessing to a day later this week)